### PR TITLE
all the little things

### PR DESCRIPTION
**Pull request by Lucky-56**
this adds:
- CoolUtil gets used more
- remove ugly pixels on week 6 (resolves #917)
- songName on SongPosBar is now relative to how long the song name is

### CODE REVIEW

**Comment by Lucky-56** [comment]
well and to  be honest, I would've been stuck anyway xD

image

**Review by isakube — APPROVED** [review]
this is a gaming

**Comment by Lucky-56** [comment]
dies, need to commit fix

File: source/PlayState.hx [hunk 1]
```diff
@@ -941,7 +933,7 @@ class PlayState extends MusicBeatState
 				songPosBar.createFilledBar(FlxColor.GRAY, FlxColor.LIME);
 				add(songPosBar);
 	
-				var songName = new FlxText(songPosBG.x + (songPosBG.width / 2) - 20,songPosBG.y,0,SONG.song, 16);
+				var songName = new FlxText(songPosBG.x + (songPosBG.width / 2) - (SONG.song.length * 5)),songPosBG.y,0,SONG.song, 16);
```
Comment (Lucky-56) [inline source/PlayState.hx]: dies here:
`(SONG.song.length * 5))` should be
`(SONG.song.length * 5)`

**Review by Lucky-56 — COMMENTED** [review]
fixed

**Comment by Lucky-56** [comment]
should I change this:

image

to this?

image

**Comment by Trayfellow** [comment]
> should I change this:
> 
> image
> 
> to this?
> 
> image

this, i need this

i'll be using this thank you very much

**Comment by Trayfellow** [comment]
gaming

**Pull request merged**